### PR TITLE
Bugfixes for the installer regarding s3 bucket

### PR DESCRIPTION
**Pull request by admiral0**
Installer was not creating the s3 bucket and was not waiting for its creation

**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**
/area installer

**What this PR does / why we need it**:

First time installation to account will fail otherwise

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: installer/util/aws.go [hunk 1]
```diff
@@ -61,6 +62,7 @@ func EnsureBucketExists(bucketName string, s3Client *s3.Client) error {
 		if err != nil {
 			return fmt.Errorf("could not create S3 bucket %s: %w", bucketName, err)
 		}
+		time.Sleep(5 * time.Second) // give AWS time to create bucket
```
Comment (fntlnz) [inline installer/util/aws.go:65]: The sdk provides a waiter for this one 

See: https://github.com/aws/aws-sdk-go/blob/v1.36.29/service/s3/waiters.go#L16

**Review by fntlnz — COMMENTED** [review]
Just added that the sdk has waiters specifically for what we need to do here

File: installer/util/aws.go (hunk 1 above)
Comment (admiral0) [reply installer/util/aws.go:65]: We use sdkv2 - could not find a way to do a waiter there.

Comment (admiral0) [reply installer/util/aws.go]: Apparently version of aws sdk v2 v0.31 has it. Migrated to v1.0.0 - possibly no more api breakage!

**Review by fntlnz — DISMISSED** [review]

**Review by fntlnz — APPROVED** [review]

**Pull request merged**